README.md

```diff
@@ -106,6 +106,14 @@ are still up-to-date.
 
 To build a GHC source distribution tarball, run Hadrian with the `sdist-ghc` target.
 
+#### Installation
+
+To install GHC artifacts, first build everything, then run the `install` target.
```

Comment:
Why is the first step needed? All required artefacts should be (re)built automatically if need be, before copying, via Shake's tracking. Users shouldn't have to remember these things.